python/openmldb/test/openmldb_client_test.py

```diff
@@ -395,6 +395,55 @@ def test_drop_table(self):
 
     def teardown_class(self):
         self.cursor.close()
+        
+class TestSQLMagicOpenMLDB:
+
+    def setup_class(self):
+        self.db = openmldb.dbapi.connect('db_test', '127.0.0.1:6181', '/onebox')
+        self.ip = openmldb.sql_magic.register(self.db)
+
+    def execute(self,magic_name,line):
+        try:
+            self.ip.run_line_magic(magic_name,line)
+            return 'ok'
+        except Exception as e:
+            raise Exception(e)
 
+    def test_magic_create_table(self):
+        try:
+            self.ip.run_cell_magic('sql','', "create table magic_table (x string, y int);")
+        except Exception as e:
+            assert False
+        assert "magic_table" in self.db.cursor().get_all_tables()
+
+        with pytest.raises(Exception):
+            assert self.execute('sql', "create table magic_table;")
+
+    def test_magic_insert(self):
```

Comment:
Since the name of the test class is `TestSQLMagicOpenMLDB`. The names of method could be simpler, like `test_insert`.